### PR TITLE
fix contirbution form script bug

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Fixed a bug that was caused in this PR - https://github.com/demisto/content/pull/17978 
where due to the fact that the contribution form filled workflow now does not check if the contribution form filled label is present, there was no validation in the script that checked if the contribution form filled label was present before checking which support type the files that were changed leading partner/community support types to automatically fail.

In addition this PR will remove necessary checks and will refer only to the labels in the pull request.

## Screenshots
image

image

image

image

### CODE REVIEW

**Comment by GuyAfik** [comment]
@ShahafBenYakir I got rid of all the un-relevant code, see example workflow above in screen shots

File: Utils/github_workflow_scripts/check_if_needs_to_fill_contribution_form.py [hunk 1]
```diff
@@ -128,75 +33,45 @@ def arguments_handler():
...
-
     pr_label_names = [label.name for label in pr.labels]
 
-    if not verify_labels(pr_label_names=pr_label_names):
+    is_contribution_form_filled_label_exist = CONTRIBUTION_FORM_FILLED_LABEL in pr_label_names
+    is_community_label_exist = COMMUNITY_LABEL in pr_label_names
+    is_partner_label_exist = PARTNER_LABEL in pr_label_names
+    is_internal_label_exist = INTERNAL_LABEL in pr_label_names
+
+    print(f'{t.cyan}Check that {CONTRIBUTION_FORM_FILLED_LABEL} label exist in PR {pr_number}')
```
Comment (ShahafBenYakir) [inline Utils/github_workflow_scripts/check_if_needs_to_fill_contribution_form.py:56]:
```suggestion
    print(f'{t.cyan}Checking if {CONTRIBUTION_FORM_FILLED_LABEL} label exist in PR {pr_number}')
```

File: Utils/github_workflow_scripts/check_if_needs_to_fill_contribution_form.py [hunk 2]
```diff
@@ -128,75 +33,45 @@ def arguments_handler():
...
+    if not is_contribution_form_filled_label_exist:
+        print(
+            f'{t.red}ERROR: Contribution form was not filled for PR: {pr_number}.\nMake sure to register your'
+            f' contribution by filling the contribution registration form in - https://forms.gle/XDfxU4E61ZwEESSMA'
+        )
+        sys.exit(1)
+
+    print(f'{t.cyan}Check that one of Community/Partner/Internal labels exist in PR {pr_number}')
```
Comment (ShahafBenYakir) [inline Utils/github_workflow_scripts/check_if_needs_to_fill_contribution_form.py:64]:
```suggestion
    print(f'{t.cyan}Checking if one of Community/Partner/Internal labels exist in PR {pr_number}')
```

**Review by ShahafBenYakir — APPROVED** [review]
Nice work!
Small edit to text

**Pull request merged**